### ISSUE TITLE
Security Concerns

### ISSUE THREAD

**Issue by ryelle** [ryelle/WP-Dashboard-SVG#6]
Are there any security concerns with using inline SVGs? This may depend on the method we choose in #1, but if anyone has any preliminary concerns, let's talk through them here.

**Comment by melchoyce** [comment]
Nope: https://wordpress.slack.com/archives/design-dashicons/p1431465120000012

:)
